src/DeckOfCards.java

```diff
@@ -0,0 +1,11 @@
+/**
+ * Created by sparatan117 on 11/28/16.
+ */
+// stores a deck of cards, vlaues of the cards, face of card, and tells if it is black.
+public class DeckOfCards {
+    int numbersOfCards;
```

Comment:
Try to work of's out of your names.  `cardCount` or `cardTotal would be what I would suggest.